### PR TITLE
Added patient notes to backend

### PR DESCRIPTION
**Pull request by edong36**
Patient notes are now stored in the database. Notes can also displayed in the patient list.

### CODE REVIEW

File: client/src/components/recordList.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ const Record = (props) => (
    <td>{props.record.height}</td>
    <td>{props.record.weight}</td>
    <td>{props.record.medications}</td>
+   <td>{props.record.notes}</td>
```
Comment (MichaelEHammond) [inline client/src/components/recordList.js:11]: This is not necessary. It should only show up on the patient's calculation page.

File: client/src/components/recordList.js [hunk 2]
```diff
@@ -89,6 +90,7 @@ export default function RecordList() {
             <th>Height</th>
             <th>Weight</th>
             <th>Medications</th>
+            <th>Notes</th>
```
Comment (MichaelEHammond) [inline client/src/components/recordList.js:93]: See above comment.

**Review by MichaelEHammond — CHANGES_REQUESTED** [review]

File: client/src/components/recordList.js (hunk 1 above)
Comment (edong36) [reply client/src/components/recordList.js]: Removed in most recent commit.

File: client/src/components/recordList.js (hunk 2 above)
Comment (edong36) [reply client/src/components/recordList.js]: Removed in most recent commit.

**Review by NamkhangNLe — COMMENTED** [review]
Notes added with no issues. Small changes are comprehensive and easy to digest. Great job!

**Review by MichaelEHammond — APPROVED** [review]

**Pull request merged**